include/swift/Runtime/Metadata.h

```diff
@@ -1745,11 +1745,10 @@ using EnumMetadata = TargetEnumMetadata<InProcess>;
 template <typename Runtime>
 struct TargetFunctionTypeMetadata : public TargetMetadata<Runtime> {
   using StoredSize = typename Runtime::StoredSize;
-
-  // TODO: Make this target agnostic
-  using Argument = FlaggedPointer<const TargetMetadata<Runtime> *, 0>;
+  using Argument = const TargetMetadata<Runtime> *;
 
   TargetFunctionTypeFlags<StoredSize> Flags;
+  TargetPointer<Runtime, const uint32_t> ParamFlags;
```

Comment:
@rjmccall Just wanted to double check if that's what you were talking about when referring to how tuple metadata is done?